extensions/src/test/java/org/pentaho/platform/web/http/api/resources/Log4jResourceTest.java

```diff
@@ -0,0 +1,90 @@
+/*!
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2017 Pentaho Corporation..  All rights reserved.
+ */
+
+package org.pentaho.platform.web.http.api.resources;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.log4j.helpers.Loader;
+import org.apache.log4j.xml.DOMConfigurator;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.core.Response;
+import java.util.Enumeration;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * Created by jnieh on 6/25/2017.
+ */
+public class Log4jResourceTest {
+
+  private static final String CONFIG = "log4j.xml";
+  private Log4jResource target = new Log4jResource();
+
+  @Test
+  public void resetLogLevel() throws Exception {
+    Logger root = LogManager.getRootLogger();
+    Level initialLevel = root.getLevel();
+    assertNotEquals( initialLevel, Level.ALL );
+    root.setLevel( Level.ALL );
+    assertEquals( Level.ALL, root.getLevel() );
+
+    Response res = target.resetLogLevel();
+    assertNotEquals( Level.ALL,  LogManager.getRootLogger().getLevel() );
```

Comment:
rather than checking that the level is not equal to ALL, wouldn't it be better to check that it's equal to the expected value?